stable/appmesh-controller/templates/deployment.yaml

```diff
@@ -32,6 +32,11 @@ spec:
             {{- if .Values.region }}
             - --aws-region={{ .Values.region }}
             {{- end }}
+            {{- if eq .Values.log.level "debug" }}
+            - --v=4
+            {{- else if eq .Values.log.level "info" }}
+            - --v=0
```

Comment:
In case the user does want to set the number, should we just pass through in an else block?: 
```
{{- else }}
- --v={{ .Values.log.level }}
{{- end }}
```

Reply:
we can add it later, I am not sure if that is useful anyways because we are not strictly following numbered logging.